### DISCUSSION TITLE
Download multiple sites to one guide.xml file

### DISCUSSION THREAD

**Discussion by lolobyte** [iptv-org/community#993] [Q&A]
Hi, is it possible to implement a function like this example?
npm run grab -- --site=example1.com; example2.com; example3.com  --output=guide.xml

**Comment by PopeyeTheSai10r** [comment]
You should be able to combine the channels from example1.com,  example2.com and  example3.com and then run the following command.

`npm run grab -- --channels=example-combined.channels.xml`

**Comment by freearhey** [comment]
https://github.com/iptv-org/epg?tab=readme-ov-file#use-custom-channel-list
